app/templates/base.html

```diff
@@ -48,6 +48,7 @@
           <div class="flash_container" id="flash_container">
             {% with messages = get_flashed_messages(with_categories=true) %}
               <script> var flashMessages = {{messages | tojson}} </script>
+              {# <script> var flashMessages = {{messages | tojson}} </script> #}  {#FIXME: SHOULD BE THIS, WHY??#}
```

Comment:
Not sure what this comment was doing. I'm guessing you figured this out?

Reply:
figured it out, removed comment line that was unused